addons/cluster-autoscaler/cluster-autoscaler.yaml

```diff
@@ -13,9 +13,6 @@
 # limitations under the License.
 
 {{ $version := "UNSUPPORTED" }}
-{{ if eq .Cluster.MajorMinorVersion "1.20" }}
-{{ $version = "v1.20.0"}}
-{{ end }}
```

Comment:
I think we discussed it before, but it might be worth bringing it up again. Isn't this going to break existing 1.20 clusters until they're not upgraded to v1.21? Upgrading to v1.21 might take some time and you can also limit how many clusters will be upgraded at the same time (if I'm not mistaken). You can also disable auto-upgrades by overriding the default config.

Shouldn't we leave this for one more version just to be sure we're not going to break something?

Reply:
Yeah good point. I had the same thought for all the Go code, but treated addons differently.

Reply:
Fixed. Restored addons to their previous state.